tests/e2e/specs/MessageStartEvent.spec.js

```diff
@@ -6,21 +6,23 @@ import { nodeTypes } from '../support/constants';
 describe('Message Start Event', () => {
   it('Can create message start event', function() {
     const messageStartEventPosition = { x: 250, y: 250 };
-    dragFromSourceToDest(nodeTypes.messageStartEvent, messageStartEventPosition);
+    dragFromSourceToDest(nodeTypes.startEvent, messageStartEventPosition);
```

Comment:
Debated putting these two lines in a function together, but couldn't find the right way.